Gruntfile.js

```diff
@@ -65,7 +105,14 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-htmlhint');
   grunt.loadNpmTasks('grunt-jscs');
   grunt.loadNpmTasks('grunt-contrib-jshint');
+  grunt.loadNpmTasks('grunt-contrib-jshint');
+  grunt.loadNpmTasks('grunt-vulcanize');
+  grunt.loadNpmTasks('grunt-contrib-copy');
+  grunt.loadNpmTasks('grunt-contrib-clean');
 
-  // set default tasks to run when grunt is called without parameters
+  // Set default tasks to run when grunt is called without parameters
   grunt.registerTask('default', ['csslint', 'htmlhint', 'jscs', 'jshint']);
+
+  // Cleans out/ folder, copies files in place and vulcanizes index.html to out/.
+  grunt.registerTask('build', ['clean', 'copy', 'vulcanize']);
```

Comment:
Does travis runs this step too? I would like travis to fail if somehow the vulcanize or other steps fail there.